pkg/nmpolicy/generate.go

```diff
@@ -93,13 +93,13 @@ func convertGeneratedStateToEnactmentConfig(nmpolicyGeneratedState nmpolicytypes
 	for captureKey, capturedState := range nmpolicyGeneratedState.Cache.Capture {
 		capturedState := nmstateapi.NodeNetworkConfigurationEnactmentCapturedState{
 			State: nmstateapi.State{
-				Raw: capturedState.State,
+				Raw: []byte(capturedState.State),
```

Comment:
isn't `capturedState.State.Raw` []byte already?

Reply:
It is but `captureState.State` is not so we have to cast it.

Reply:
Ah of course